File: frontend/emergencyMS-web/src/app/models/types/TypeEntity.ts
import { EmergencySubType } from "./EmergencySubType";
import { EmergencyTypeEntity } from "./EmergencyTypeEntity";
import { RegionType } from "./RegionType";
import { SourceType } from "./SourceType";

export interface TypeEntity {
    types: EmergencyTypeEntity[];
    subTypes : EmergencySubType[];
    regionTypes: RegionType[];
    sourceTypes: SourceType[];
}

File: backend/EmengencyMS.API/Domain/Entities/TypeEntity.cs
using Domain.Entities.Types;

namespace Domain.Entities;

public class TypeEntity
{
    public IEnumerable<EmergencyType> Types { get; set; }
    public IEnumerable<EmergencySubType> SubTypes { get; set; }
    public IEnumerable<Region> Regions { get; set; }
    public IEnumerable<SourceType> SourceTypes { get; set; }
}